core/src/main/java/bio/terra/pearl/core/dao/participant/EnrolleeRelationDao.java

```diff
@@ -99,4 +99,18 @@ public void deleteByStudyEnvironmentId(UUID studyEnvironmentId) {
                         .execute()
         );
     }
+
+    public List<EnrolleeRelation> findAllFamilyRelationshipsByEnrolleeIdOrTargetIdAndFamilyId(UUID enrolleeId, UUID familyId) {
```

Comment:
how about `findAllRelationsByFamilyAndEitherEnrollee()` for something a bit shorter